core/settings.py

```
@@ -0,0 +1,741 @@
+            "url": "https://brooksjr.com",
+            "email": "terry@brooksjr.com",
+        },
+        "EXTERNAL_DOCS": {
+            "url": "https://github.com/Terry-BrooksJr/the-dozens-django",
+            "description": "GitHub Repository and full API documentation Hub",
+        },
+    }
+    REST_FRAMEWORK_EXTENSIONS = values.DictValue({"DEFAULT_CACHE_ERRORS": False})
+    #!SECTION End - REST API.SWAGGER DOCUMENTATION SETTINGS
+
+    #  SECTION Start - GraphQL Settings (Graphene-Django)
```

Pylint (3.3.7)

[warning] SECTION Start - GraphQL Settings (Graphene-Django)

(W0511)